### PR TITLE
Mock data for ticket and event

### PR DESCRIPTION
**Pull request by AlexanderHansson4225**
Created the ability for tickets to have types such as lunch or banquet. Created mockdata for the corresponding tickets (both for events and tickets) Also tests and a migrate.

### CODE REVIEW

File: Nexpo/Repositories/TicketRepository.cs
Comment (emil-ronkainen) [inline Nexpo/Repositories/TicketRepository.cs:60]: I am uncertain on how this will work in practice. From what I can tell, when doing `Select` on an enum, the `FirstOrDefaultAsync` will fall back on the first value of the enum as default, i.e. we return a `CompanyEvent` on invalid requests. Adding a default "fallback" or "failure" entry to the enum and syncing this with the corresponding frontend calls might resolve this.

Assuming this is only called with valid `ticketId` arguments it's probably ok to ignore this, but for testing/debugging purposes it might be nice to keep the errors verbose

**Review by emil-ronkainen — CHANGES_REQUESTED** [review]
Some topics for discussion but looking good overall!

**Review by IsaRandsalu — COMMENTED** [review]
I think it looks good :)
I'm not so familiar with how the backend syncs with the frontend, so maybe you should take a look at Emil's comments before merging :)

**Pull request merged**
